search/search_repository_whitebox_test.go

```diff
@@ -175,10 +176,10 @@ func (s *searchRepositoryWhiteboxTest) TestSearchByText() {
 				s.T().Fatal("Couldnt create test data")
 			}
 
-			defer wir.Delete(ctx, createdWorkItem.SpaceID, createdWorkItem.ID, s.modifierID)
+			//defer wir.Delete(ctx, createdWorkItem.ID, s.modifierID)
```

Comment:
yes, this line was initially commented out to verify that it was not required to keep the DB clean, and it can now be removed.